server/athenian/api/models/state/models.py

```diff
@@ -640,3 +640,48 @@ class DashboardChartGroupBy(create_time_mixin(created_at=True, updated_at=True),
     DashboardChartGroupBy.jira_issue_types,
     DashboardChartGroupBy.jira_labels,
 )
+
+
+class InstallationProgress(create_time_mixin(created_at=False, updated_at=False), Base):
+    """The group by configured for a dashboard chart."""
+
+    __tablename__ = "installation_progress"
+
+    account_id = Column(
+        Integer(),
+        ForeignKey("accounts.id", name="fk_installation_progress_account"),
```

Comment:
This FK assumes that there must be an Athenian account associated with the record. This is not the case for GitHub installations: they usually start without Athenian account. So we won't be able to insert most of the records here, unless we force Athenian account creation from MD side.

Reply:
ok removed

Reply:
So what's your plan for the `account_id` column? Is it still an Athenian account ID or GitHub account ID?

Reply:
this have to be a Github ID as it is in the Event